internal/selfmonitor/config/testdata/config.yaml

```diff
@@ -3,7 +3,8 @@ global:
     evaluation_interval: 30s
 alerting:
     alertmanagers:
-        - static_configs:
+        - scheme: https
```

Comment:
why do we set the default to `https` just to directly after it only use hosts with `http`?